toolkit/build.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
```

Comment:
I would consider renaming the script to `build_package.sh` to avoid the ambiguity on _what_ it is building.
Additionally, if we decide to add more such scripts in the future to build image/ toolchain, etc. we would not need to rename this script.

Reply:
I feel i might rename it to `pkgbld` so that it is not too much typing.